spring-cloud-stream-binders/spring-cloud-stream-binder-kafka/src/test/java/org/springframework/cloud/stream/binder/kafka/KafkaBinderTests.java

```diff
@@ -217,17 +220,17 @@ public void testCustomPartitionCountDoesNotOverrideModuleCountAndConcurrencyIfSm
 
 		byte[] ratherBigPayload = new byte[2048];
 		Arrays.fill(ratherBigPayload, (byte) 65);
-		KafkaTestBinder binder = (KafkaTestBinder) getBinder();
+		KafkaTestBinder binder = getBinder();
 
 
 		DirectChannel moduleOutputChannel = new DirectChannel();
 		QueueChannel moduleInputChannel = new QueueChannel();
-		Properties producerProps = new Properties();
-		producerProps.put(BinderPropertyKeys.MIN_PARTITION_COUNT, "5");
-		producerProps.put(BinderPropertyKeys.NEXT_MODULE_CONCURRENCY, "6");
-		Properties consumerProps = new Properties();
-		consumerProps.put(BinderPropertyKeys.MIN_PARTITION_COUNT, "5");
-		consumerProps.put(BinderPropertyKeys.CONCURRENCY, "6");
+		KafkaProducerProperties producerProps = new KafkaProducerProperties();
+		producerProps.setPartitionCount(5);
+		producerProps.setPartitionCount(6);
```

Comment:
this overrides the call on the line above... previously these were 2 separate props: `MIN_PARTITION_COUNT` and `NEXT_MODULE_CONCURRENCY`